consensus/istanbul/backend/backend.go

```diff
@@ -648,15 +648,12 @@ func (sb *Backend) validatorRandomnessAtBlockNumber(number uint64, hash common.H
 	if number > 0 {
 		lastBlockInPreviousEpoch = number - istanbul.GetNumberWithinEpoch(number, sb.config.Epoch)
 	}
-	header := sb.chain.GetHeaderByNumber(lastBlockInPreviousEpoch)
-	if header == nil {
```

Comment:
Why remove this?

Reply:
I assume `sb.chain.CurrentHeader` can never be nil, but you are right, I might as well leave the check in there